pypeit/pypeit.py

```diff
@@ -689,6 +785,69 @@ def _extract_one(self):
 
         return sciimg, sciivar, skymodel, objmodel, ivarmodel, outmask, sobjs, vel_corr
 
+    def _extract_std(self):
```

Comment:
This extract_std method should be removed. It  is just duplicating a ton of code and as such will be difficult to maintain. The extraction steps are almost exactly  the same for the standard as for normal science objects, and any differences can simply be handled with a std = True being passed into extract_one.   

I know how much you like wrappers, so if you want you can have extract_std point to extract_one with std = True. I can take it from there to make sure things are done sensibly for the standard. Basically there is just going to be a std flag passed into objfind to make the objmask larger (maybe), and then something will be passed into local_sksyub (for which I already have hooks).

Joe